pages/tasks/task_details.py

```diff
@@ -11,6 +11,7 @@
 class TaskDetailsPage(Base):
 
     _name_locator = (By.CSS_SELECTOR, '.content-container > h1')
+    _team_locator = (By.CSS_SELECTOR, '.three-fields > a')
```

Comment:
I'd be interested in updating the app but I'm still new to HTML and CSS (though adding an ID should be easy). For now, I'm trying to select based on "a href contains 'team'" and I'm struggling. It seems to be quite a bit harder than it should be.

I was thinking something like this but it doesn't seem to be valid (I can't find very good documentation on complicated CSS selectors):
    _team_link_locator = (By.CSS_SELECTOR, 'a[href]*=team')

I though maybe xpath would work too but I can't figure out how to do it without finding all the a elements, getting the href attribute and searching for 'team'. Do you have a suggestion about the best approach (CSS, xpath, other)?